### ISSUE TITLE
Hand position descriptions are a little different than expected 

### ISSUE THREAD

**Issue by terracoda** [phetsims/john-travoltage#303]
I just listened to the sim on Master (September 18, 2018)

It is possible that some errors to the Hand position descriptions were introduced when work on #282, #285 and #291 were going on.

The main thing missing from the position descriptions is "Last stop." from positions -15 and 15, but I am also not 100% sure the logic for appending Region Descriptions [RD] is happening correctly. I would need to test a bit more to very correct changes in directions, Landmarks and Region Descriptions. are all being combined as was intended.

The position descriptions should be like this chart:
![jt-hand-positions](https://user-images.githubusercontent.com/1902116/45692278-f2bd8400-bb34-11e8-96ce-db3e643a8e5a.png)

See also comments https://github.com/phetsims/john-travoltage/issues/282#issuecomment-395744281 and https://github.com/phetsims/john-travoltage/issues/282#issuecomment-395748501

I'll mark this issue as a bug and for discussion in Office Hours, so we can set a priority.

**Comment by terracoda** [comment]
@emily-phet, just tagging you, so you are aware of this issue and issue #302. I added both to today's office hours.

I think this issue should be fixed before re-publishing with sound, but #302 doesn't need to slow down a new release of this sim.

**Comment by jessegreenberg** [comment]
Thanks for pointing this out @terracoda, very likely caused by #285. When exploring the circular slider, the "Last Stop" strings were removed. Ill add them back

**Comment by jessegreenberg** [comment]
@terracoda the "Last Stop" string was added back. The other part of this issue is to verify that the rest of the behavior is correct, at meeting today we can discuss the best way to do this.

**Comment by terracoda** [comment]
Thanks @jessegreenberg, "Last stop" sounds great, and I also re-checked the positions and the changes in direction and all is working as should be! It seems, just the "Last stop" string was missing.

Closing this issue.
